src/mediaplayer.cs

```diff
@@ -1272,6 +1285,30 @@ interface MPNowPlayingInfoCenter {
 		[Internal]
 		[Field ("MPNowPlayingInfoPropertyCurrentLanguageOptions")]
 		NSString PropertyCurrentLanguageOptions { get; }
+
+		[iOS (9,3)]
```

Comment:
did you test it on an iOS 9.3 device ? if so please add the information in the commit message